src/libstd/memchr.rs

```diff
@@ -209,7 +209,7 @@ mod fallback {
         let end_align = (ptr as usize + len) & (usize_bytes - 1);
```

Comment:
Preexisting, but this uses `size_of` to calculate `usize_bytes`, but alignment of usize on a system is not `size_of::<usize>` but rather `align_of::<usize>`? Seems like a pessimisation?